test-case/check-audio-equalizer.sh

```diff
@@ -44,7 +44,7 @@ func_test_eq()
     local id=$1
     local conf=$2
 
-    [ ! -f $conf ] && dloge "$conf not exist" && exit 1
+    [ ! -f $conf ] && die "$conf not exist"
```

Comment:
Not in this PR now (too much change) but in the future  try to stick to the codestyle `test_something || die` and `assert_something || die` codestyle that is standard, reads like plain english and is "more obviously" compatible with `set -e`

For instance here it would be:
```shell
 [ -f $conf ] || die "$conf not exist"
```

Reply:
thanks, sure. will submit a new PR later to correct this kind of code